packages/store/src/store.ts

```diff
@@ -51,6 +59,8 @@ export class Store {
       spaceId,
       new Space(spaceId, this.doc, this.awareness, this.idGenerator)
     );
+    this._indexer.onCreateSpace(spaceId);
```

Comment:
What should we do after the space is updated afterward?

Reply:
seem we cannot observe root map change (doc update event / abstract type event)